cico_run_tests.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+
+# This file is supposed to be executed by CICO build (ci.centos.org/view/Devtools/)
+#   1. Run tests for fabric8-planner
+#   2. Create snapshot with fabric8-ui's master and fabric8-planner's current changes
+
+# Show command before executing
+set -x
+
+# Exit on error
+set -e
+
+# Source environment variables of the jenkins slave
+# that might interest this worker.
+if [ -e "jenkins-env" ]; then
+  cat jenkins-env \
+    | grep -E "(JENKINS_URL|DEVSHIFT_USERNAME|DEVSHIFT_PASSWORD|GIT_BRANCH|GIT_COMMIT|BUILD_NUMBER|ghprbSourceBranch|ghprbActualCommit|BUILD_URL|ghprbPullId)=" \
+    | sed 's/^/export /g' \
+    > /tmp/jenkins-env
+  source /tmp/jenkins-env
+fi
+
+# We need to disable selinux for now, XXX
+/usr/sbin/setenforce 0
+
+# Get all the deps in
+yum -y install docker make git
+service docker start
+
+# Build builder image
+cp /tmp/jenkins-env .
+
+mkdir -p fabric8-ui-dist
+# Build fabric8-planner image
+docker build -t fabric8-planner-builder -f Dockerfile .
+# User root is required to run webdriver-manager update.
+# This shouldn't be a problem for CI containers
+docker run --detach=true --name=fabric8-planner -v $(pwd)/fabric8-ui-dist:/home/fabric8/fabric8-planner/fabric8-ui-dist:Z --cap-add=SYS_ADMIN -t fabric8-planner-builder
```

Comment:
`docker run` returns the ID of the container being run. It would be nice if you don't specify the name `fabric8-planner` but instead use the ID below in `docker exec fabric8-planner`.